metaflow/runtime.py

```diff
@@ -134,6 +134,10 @@ def __init__(
         # finished.
         self._control_num_splits = {}  # control_task -> num_splits mapping
 
+        # to allow the current singleton access within Tasks (which are in subprocesses)
+        os.environ["MAX_WORKERS"] = str(max_workers)
```

Comment:
I would make these names a bit more specific to avoid the possibility of conflicts.

Reply:
Good point, will change